[18.0][FIX] l10n_es_verifactu_oca: Change field name reversal_move_id -> reversal_move_ids

Andrii9090-tecnativa opened the pull request:
In this commit (https://github.com/odoo/odoo/commit/26d57934f7a656fe5267d7931e0601c43a18d270), the field 'reversal_move_id' in the model 'account.move' was renamed to 'reversal_move_ids'.

@moduon

pedrobaeza reviewed (commented):
Gracias por el parche. No cambies por favor la versión del módulo a mano. Se cambia al fusionar con el bot. Así además el PR queda más limpio solo con el cambio importante.

pedrobaeza commented:
Los cambios en el README.rst e index.html han permanecido y no deberían.

Commit:
[FIX] l10n_es_verifactu_oca: Change field name reversal_move_id -> reversal_move_ids

In this commit (https://github.com/odoo/odoo/commit/26d57934f7a656fe5267d7931e0601c43a18d270),
the field 'reversal_move_id' in the model 'account.move' was renamed to 'reversal_move_ids'.

pedrobaeza reviewed (approved):
/ocabot merge patch
